pyproject.toml

```diff
@@ -1,92 +1,102 @@
 [project]
 name = "brainglobe-workflows"
-authors = [{name = "BrainGlobe developers", email= "a.felder@ucl.ac.uk"}]
-description = "A place to keep scripts to use as benchmarks, user-examples end-to-end tests and for tutorials"
-readme = "README.md"
-requires-python = ">=3.8.0"
-dynamic = ["version"]
-dependencies = [
-    "pooch",
-    "cellfinder-core"
+authors = [
+    { name = "Adam Tyson", email = "code@adamltyson.com" },
+    { name = "BrainGlobe developers", email = "code@adamltyson.com" },
+    { name = "Christian Niedworok" },
+    { name = "Charly Rousseau" },
 ]
-license = {text = "BSD-3-Clause"}
-
+description = "A collection of end-to-end data analysis workflows executed using BrainGlobe tools."
+readme = "README.md"
+license = { file = "LICENSE" }
+requires-python = ">=3.9"
 classifiers = [
-    "Development Status :: 2 - Pre-Alpha",
-    "Programming Language :: Python",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Operating System :: OS Independent",
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
     "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python",
+    "Topic :: Scientific/Engineering :: Image Recognition",
+]
+dependencies = [
+    "brainreg>=1.0.0",
+    "cellfinder-core>=0.2.4,<1.0.0",
+    "configobj",
+    "fancylog>=0.0.7",
+    "imio",
+    "brainglobe-utils>=0.2.5",
+    "multiprocessing-logging>=0.3.4",
+    "natsort",
+    "numpy",
+    "pandas",
+    "packaging",
+    "pooch",
+    "scikit-image",
+    "tifffile",
+    "tqdm",
 ]
+dynamic = ["version"]
 
-[project.urls]
-"Homepage" = "https://github.com/brainglobe/brainglobe-scripts"
-"Bug Tracker" = "https://github.com/brainglobe/brainglobe-scripts/issues"
-"Documentation" = "https://github.com/brainglobe/brainglobe-scripts"
-"Source Code" = "https://github.com/brainglobe/brainglobe-scripts"
-"User Support" = "https://github.com/brainglobe/brainglobe-scripts/issues"
+[project.scripts]
+cellfinder = "brainglobe_workflows.main:main"
 
 [project.optional-dependencies]
 dev = [
-  "pytest",
-  "pytest-cov",
-  "coverage",
-  "tox",
-  "black",
-  "mypy",
-  "pre-commit",
-  "ruff",
-  "setuptools_scm",
+    "black",
+    "pytest-cov",
+    "pytest",
+    "gitpython",
+    "coverage>=5.0.3",
+    "pre-commit",
+    "setuptools_scm",
 ]
+napari = ["napari[pyside2]", "brainglobe-napari-io", "cellfinder-napari<1.0.0"]
```

Comment:
I'm not entirely sure - this seems to have been an artefact of cellfinder since before we moved to `pyproject.toml` too: https://github.com/brainglobe/cellfinder/pull/247/files#

Looking at [brainreg](https://github.com/brainglobe/brainreg/blob/main/pyproject.toml, we're shipping `napari` with a backend (albeit Qt5) there too. As an experiment I'll change this to ship with Qt5 for consistency, but my answer to "why ship a backend" is an unsatisfactory appeal to tradition :sweat_smile:

Reply:
Cool - Opened https://github.com/brainglobe/BrainGlobe/issues/53 for this!

Reply:
I think we went with `pyside2` due to some licensing issues, but TBH I think we're just ignoring that for now.

Don't we need to ship some back-end, otherwise there's an extra installation step for users?

Reply:
> Don't we need to ship some back-end, otherwise there's an extra installation step for users?

Yeah if we don't ship a backend it'll be an extra step. 

Although this might mean that we're inadvertently overwriting their backend choice, if they have one installed already. EG If they have pyside installed and we then force-install napari[pyqt5], I'm not sure what that'll do to their env/machine